public/apps/configuration/utils/display-utils.tsx

```diff
@@ -43,6 +43,10 @@ export function displayObject(object: object | undefined) {
   return !isEmpty(object) ? JSON.stringify(object) : '--';
 }
 
+export function displayMap(map: object) {
+  return map && Object.keys(map).length !== 0 ? JSON.stringify(map, null, 2) : '--';
+}
```

Comment:
It looks just `displayObject` but just pretty to me. Why not an argument `pretty` to the existing `displayObject`?

Reply:
This was originally for compliance setting. Let me remove it for now and refactor together with compliance settings.